
# HG changeset patch
# User mchinnathamb
# Date 1540559106 -19800
# Node ID fff95315a49f58aa7ff37f7434978d554df59565
# Parent  ad4ea3c9b42b6fe3f83ab12e3aff0f2138ff6fd0
8211714: Need to update vm_version.cpp to recognise VS2017 minor versions
Reviewed-by: dholmes

diff -r ad4ea3c9b42b -r fff95315a49f src/share/vm/runtime/vm_version.cpp
--- a/src/share/vm/runtime/vm_version.cpp	Wed Sep 26 06:28:48 2018 -0700
+++ b/src/share/vm/runtime/vm_version.cpp	Fri Oct 26 18:35:06 2018 +0530
@@ -218,10 +218,16 @@
         #define HOTSPOT_BUILD_COMPILER "MS VC++ 12.0 (VS2013)"
       #elif _MSC_VER == 1900
         #define HOTSPOT_BUILD_COMPILER "MS VC++ 14.0 (VS2015)"
+      #elif _MSC_VER == 1911
+        #define HOTSPOT_BUILD_COMPILER "MS VC++ 15.3 (VS2017)"
       #elif _MSC_VER == 1912
         #define HOTSPOT_BUILD_COMPILER "MS VC++ 15.5 (VS2017)"
       #elif _MSC_VER == 1913
         #define HOTSPOT_BUILD_COMPILER "MS VC++ 15.6 (VS2017)"
+      #elif _MSC_VER == 1914
+        #define HOTSPOT_BUILD_COMPILER "MS VC++ 15.7 (VS2017)"
+      #elif _MSC_VER == 1915
+        #define HOTSPOT_BUILD_COMPILER "MS VC++ 15.8 (VS2017)"
       #else
         #define HOTSPOT_BUILD_COMPILER "unknown MS VC++:" XSTR(_MSC_VER)
       #endif

